x-pack/solutions/observability/plugins/synthetics/server/routes/monitor_cruds/project_monitor/add_monitor_project.ts

```diff
@@ -84,9 +86,9 @@ export const addSyntheticsProjectMonitorRoute: SyntheticsRestApiRouteFactory = (
   },
 });
 
-export const REQUEST_TOO_LARGE = i18n.translate('xpack.synthetics.server.project.delete.toolarge', {
+export const REQUEST_TOO_LARGE = i18n.translate('xpack.synthetics.server.project.delete.request', {
   defaultMessage:
-    'Delete request payload is too large. Please send a max of 250 monitors to delete per request',
+    'Delete request payload is too large. Please send a max of 250 browser monitors per request.',
```

Comment:
There isn't any mention of the lightweight limit which could theoretically be hit, perhaps we should have a separate message for lightweight monitors and conditionally put the appropriate one in the message body above?